src/System.Reflection.TypeExtensions/src/System.Reflection.TypeExtensions.builds

```diff
@@ -10,6 +10,9 @@
     <Project Include="System.Reflection.TypeExtensions.csproj">
       <TargetGroup>netcore50aot</TargetGroup>
     </Project> -->
+    <Project Include="System.Reflection.TypeExtensions.csproj">
```

Comment:
Do we really need a different build for uap101aot? Can we use the same build for both CoreCLR and CoreRT?